Cms_frontend/src/css/index.css

```diff
@@ -0,0 +1,118 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: montserrat, sans-serif;
+}
+
+input, button {
+  appearance: none;
+  background: none;
+  border: none;
+  outline:none;
+}
+
+.App {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #1D1F20
+}
+
+form {
+  display: block;
+  position: relative;
+}
+
+form:after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: -5px;
+  left: -5px;
+  right: -5px;
+  bottom: -5px;
+  z-index: 1;
+  /* background-color: #3A3E40; */
+}
+
+form .form-inner {
+  position: relative;
+  display: block;
+  /* background-color: #232526; */
+  padding: 30px;
+  z-index: 2;
+}
+
+form .form-inner h2 {
+  text-align: center;
+  color: #F8F8F8;
+  font-size: 28px;
+  font-weight: 500;
+  margin-bottom: 100px;
+}
+
+form .form-inner .form-group {
+  display: block;
+  width: 300px;
+  margin-bottom: 15px;
+}
+
+.form-inner .form-group label {
+  display: block;
+  color: #C4C4C4;
+  font-size: 12px;
+  margin-bottom: 5px;
+  transition: 0.4s;
+}
+
+.form-inner .error {
+  display: block;
+  text-align: center;
+  color: #b62e2e;
+  padding: 0px 0px 15px;
+}
+
+.form-inner .form-group:focus-within label {
+  color: #FFFFFF;
+}
+
+form .form-inner .form-group input {
+  display: block;
+  width: 100%;
+  padding: 10px 15px;
+  background-color: #C4C4C4;
+  border-radius: 8px;
+  transition: 0.4s;
+}
+
+form .form-inner .form-group input:focus {
+  box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.2);
+}
+
+form .form-inner input[type="submit"] {
+  display: inline-block;
+  padding: 10px 15px;
+  border-radius: 8px;
+  background-color: #F8F8F8;
+  background-size: 200%;
+  background-position: 0%;
+  transition: 0.4s;
+  cursor: pointer;
+  
+}
+
+form .form-inner .MuiButtonBase-root {
+  margin-top: 8px;
+ }
+
+img {
+  display: block;
+  width: auto;
+  height: auto;
+  max-width: 200px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 100px;
+}
```

Comment:
all the css code should be moved into a styled components container within your login page. This is because we dont want these css changes to be global.

Can check out Hanyuan's code to check out how he handles his css